perses/rjmc/topology_proposal.py

```diff
@@ -534,6 +592,41 @@ def _assign_ring_ids(molecule, max_ring_size=10):
             atom.SetIntType(ring_as_base_ten)
         return id_molecule
 
+    @staticmethod
+    def _assign_distance_ids(old_mol, new_mol, distance=0.3):
```

Comment:
is 0.3A just the one that happened to work?

Reply:
yes --- it can be changed, but seemed reasonable to me. I'm happy to change it though

Reply:
In fact it's not going to work for Jnk1 as the ligands are not that closely aligned...